sdk/storage/storage-blob/src/models.ts

```diff
@@ -325,3 +327,59 @@ export enum StorageBlobAudience {
    */
   DiskComputeOAuthScopes = "https://disk.compute.azure.com/.default",
 }
+
+/**
+ * Abstract representation of a poller, intended to expose just the minimal API that the user needs to work with.
+ */
+export declare interface PollerLike<TState extends PollOperationState<TResult>, TResult> {
```

Comment:
What is the difference between `export interface` and `export declare interface`?

Reply:
I copied it from the d.ts file which has the `declare` keyword. I removed it in [df286cc](https://github.com/Azure/azure-sdk-for-js/pull/22753/commits/df286ccde7ec54bf46d572f44ca1440c07953d5d).